opentelemetry.html.md.erb

```diff
@@ -64,32 +80,101 @@ otlp:
   compression: snappy
 ```
 
-### <a id='multiple-exporters'></a> Multiple exporters
+## <a id='processor-config'></a> Processor configuration
 
-The OTel Collector supports defining multiple exporters, each with its own configuration. For example, you can define
-two OTLP gRPC exporters that egress metrics to different destinations.
+Processors are used at various stages of a pipeline to pre-processes data before it is exported. By default, no processors are enabled. The order of the processors in a pipeline determines the order of the processing operations that the Collector applies to the signal. 
+
+Refer to the individual processor [documentation](https://github.com/loggregator/otel-collector-release/tree/main/src/otel-collector/vendor/go.opentelemetry.io/collector/processor) for more information.
```

Comment:
I have a couple problems with this link:
- It's currently pointing to a fork of the otel-collector-release repo.
- If we're going to point at a file in vendor then I would rather point at the source.
- In general, I prefer docs links to GitHub links.

Could it be replaced with something else? For example, https://github.com/loggregator/otel-collector-release/tree/main/src/otel-collector/vendor/go.opentelemetry.io/collector/processor?

Reply:
Whoops! Wrong link. I meant https://opentelemetry.io/docs/collector/configuration/#processors above